src/gene/query.py

```diff
@@ -424,10 +424,9 @@ def _create_concept_mapping(
                 raise ValueError(err_msg) from e
 
             system = NAMESPACE_TO_SYSTEM_URI.get(source, source)
-            code_ = concept_id.upper() if source == NamespacePrefix.HGNC else source_id
 
             return ConceptMapping(
-                coding=Coding(code=code(code_), system=system), relation=relation
+                coding=Coding(code=code(concept_id), system=system), relation=relation
```

Comment:
@jsstevenson is it fine to just use the `concept_id` as the CURIE? Technically, HGNC uses uppercase. I might be overthinking things with mappings

Reply:
Oh, maybe we just need to look at updating our `NamespacePrefix` member values in the future?

Reply:
Yeah, I think that'd be simplest. 

Realistically there is cannot be a "true" CURIE prefix for a given source (identifiers.org was supposed to be the solution, but then it has its own problems so people made the bioregistry, but it has problems and so on and so on), which is why Matt was talking about some kind of local prefix expansions document. Some sources obviously have their own personal opinions about what prefixes you use for them, but then it begs the question of what to do if there's a community standard which differs from that. We should definitely look at updating our own prefix mapping where it makes sense though.

Reply:
@jsstevenson Ah okay, maybe we wait for Matt (and others) to provide guidance then and update our `NamespacePrefix` values whenever that discussion happens?